data/json/uncraft/generic.json

```
@@ -2325,7 +2325,7 @@
     "result": "laptop",
     "type": "uncraft",
     "skill_used": "electronics",
-    "time": "1 h",
+    "time": "15 m",
```

```suggestion
    "time": "20 m",
```
15 feels just a *little* too fast, and 20 is essentially the next fraction/quotient up (1/3 instead of 1/4)